pyscf/adc/test/test_uadc/test_ee_outcore_H2O.py

```diff
@@ -105,9 +106,10 @@ def test_ee_adc3(self):
         self.assertAlmostEqual(e[3],0.3790532775, 6)
 
         self.assertAlmostEqual(p[0],0.00000000, 6)
-        self.assertAlmostEqual(p[1],0.02714686, 6)
+        self.assertAlmostEqual(p[1],0.02702943, 6)
```

Comment:
I noticed that several tests are modified, with slightly different results. Is the default initial guess changed?

Reply:
This PR added contributions of third-order single excitation amplitudes to the RDMs and oscillator strengths, which were previously not included. Thus, reference test values had to be updated.